src/Contract/Operation/Averagesable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract\Operation;
+
+use loophp\collection\Contract\Collection;
+
+/**
+ * @template TKey
+ * @template T
+ */
+interface Averagesable
+{
+    /**
+     * Compute the averages.
```

Comment:
the doc block here should match the description in the `api.rst`

Reply:
Done